src/Target.cpp

```diff
@@ -52,7 +52,7 @@ static void cpuid(int info[4], int infoType, int extra) {
 #endif
 }
 
-Target get_host_target() {
+Target calculate_host_target() {
```

Comment:
Should this be moved to an anonymous namespace now that it's not exposed? It's currently in Halide::